san/batch.py

```diff
@@ -1,6 +1,7 @@
+import san.sanbase_graphql
 from san.query import get_gql_query
 from san.graphql import execute_gql
-from san.transform import transform_query_result
+from san.transform import transform_query_result, V2_METRIC_QUERIES
```

Comment:
I do not like having to import `V2_METRIC_QUERIES` from `san.transform` - I would never search for it there in the first place. We can have a module with appropriate name that could hold this list?

Reply:
You're right, I was going to ask you that.